performance_regression_row_level_repair_test.py

```diff
@@ -0,0 +1,483 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2019 ScyllaDB
+
+import random
+import string
+import time
+
+from sdcm.tester import ClusterTester
+
+# performance_regression_row_level_repair_test.py
+from sdcm.utils.common import measure_time, retrying
+
+
+class PerformanceRegressionRowLevelRepairTest(ClusterTester):
+    """
+    Test Scylla performance regression with cassandra-stress.
+
+    """
+
+    KEYSPACE_NAME = "ks"
+    TABLE_NAME = "cf"
+    INT_COLUMNS = 20  # TODO: 99
```

Comment:
what are all those TODO ?, we want them or not ? if not, please remove them

Reply:
removed.